frontend/src/lib/interval.ts

```diff
@@ -1,28 +1,36 @@
 import { _ } from "../i18n";
 
-export type Interval = "year" | "quarter" | "month" | "week" | "day";
+export type Interval =
+	| "year"
```

Comment:
seems these lines are indented with tabs and not spaces - I noticed `ts` was missing from the pre-commit config for prettier, I just fixed that on main - can you rebase and re-run pre-commit to switch to spaces here as well?

Reply:
I noticed the formatting changes and just assumed it was covered pre-commit (maybe incrementally changes the style or something)

More than happy to rebase :+1: